script/Arcueid/Day 7/07_00_ARC07_3_1_3.txt

```diff
@@ -239,7 +239,7 @@ and are allowed to exercise their will as members of the aristocracy."
 [sha:b153daae1e2e7e068f28df2696cc0bb2cf459292]{
 -- Page 592, Offset 18057.
 -- 「奴隷じゃないってコト。
-"I mean that they are no longer slaves.
+"It means that they are no longer slaves.
```

Comment:
She's clarifying something that she herself said, so I think keep `I`

Reply:
Hmm, personally when I hear people clarifying something they would refer to the statement, and "I mean" would be past tense to "I meant xyz" but might just be me.

Reply:
I prefer 'it means'

Reply:
I agree with Yusuke and exode..